src/index.svelte

```diff
@@ -40,10 +40,12 @@
   })
 
   const cleanEvents = () => {
-    document.removeEventListener('mousemove', onMouseMove)
-    document.removeEventListener('mouseup', onDragStopped)
-    document.removeEventListener('touchmove', onTouchMove)
-    document.removeEventListener('touchend', onDragStopped)
+    if (document) {
```

Comment:
You might want to even do `typeof document !== 'undefined'` actually because accessing the variable could even fail.

Reply:
Cheers, amended the commit